### ISSUE TITLE
Cityscapes training settings

### ISSUE THREAD

**Issue by DdeGeus** [WeilunWang/semantic-diffusion-model#5]
Thanks for your interesting work!

Could you provide the exact training settings/arguments that you use for the Cityscapes experiments? For instance: do you use the same learning rate/batch size?
